### ISSUE TITLE
style: 404 page looks wrong

### ISSUE THREAD

**Issue by MortenHofft** [gbif/hp-grscicoll#25]
https://grscicoll.hp.gbif-staging.org/nonsense

This is subjective, but the 404 page looks wrong to me

<img width="670" alt="Screenshot 2023-09-22 at 09 02 26" src="https://github.com/gbif/hp-grscicoll/assets/6759267/b5e7afa7-fc29-4cc0-8165-85158933d103">

**Comment by ManonGros** [comment]
yes, I will remove the image, unless @javiesgm think we could do something with it?

**Comment by javiesgm** [comment]
On it!

**Comment by ManonGros** [comment]
Thanks @javiesgm!
